specs/credentialmanagement/index.src.html

```diff
@@ -821,9 +867,10 @@ <h4 id="interfaces-request-options">
 
     <pre>
       navigator.<a idl>credentials</a>.<a idl lt="get()">get</a>({
-        "<a idl>types</a>": [ "federated" ],
-        "<a idl>options</a>": {
-            "<a idl>providers</a>": [ "https://identity.example.com/" ]
+        "<a idl>filters</a>": {
+            "federated" : {
+                "<a idl>providers</a>": [ "https://identity.example.com/" ]
```

Comment:
Is your concern around losing the `types` property or the structure of the filters? To me this seems pretty easy to use (and understand) for anyone that just wants to use the built-in `Credential` types. Two examples explains it all.

For developers wanting to create custom types there is a lot more to wrap your ahead around but that should be expected.